pkg/cmd/roachtest/roachstress-ci.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script spawns a Roachtest-stress CI job. You need a TeamCity token:
+#
+# https://www.jetbrains.com/help/teamcity/managing-your-user-account.html#Managing+Access+Tokens
+#
+# which the script assumes is available from the TEAMCITY_TOKEN environment variable.
+
+if [ -z "${BASH_VERSINFO}" ] || [ -z "${BASH_VERSINFO[0]}" ] || [ "${BASH_VERSINFO[0]}" -lt 4 ]; then
+  echo "This script requires Bash version >= 4"
+  echo "On OSX, 'brew install bash' should do the trick."
+  exit 1
+fi
+
+if [ ! -v TEAMCITY_TOKEN ]; then
```

Comment:
Here and below we test if some variables are set.  For variables where we _require_ a value, we may want to explicitly check that they are non-empty, since a variable can be set but empty.